static/dream_web/index.css

```diff
@@ -1,61 +1,274 @@
+}
+.config-container label{
+    font-size: 14px;
+    font-weight: 500;
+    margin-bottom: 10px;
+}
+
+.config-container input, .config-container select{
+    width: 100%;
+    display: block;
+    text-align: right;
+    border: solid 2px var(--field_border);
+    border-radius: 7px;
+    padding: 5px;
+    color: var(--field_font);
+    font-weight: 600;
+    font-size: 14px;
+    transition: ease all 0.2s;
+}
 
-    border-radius: 20px 0px 0px 20px;
-    padding: 5px 10px 5px 10px;
-    border: 1px solid black;
-    border-right: none;
+.config-container input:focus, .config-container select:focus{
+    border-color: var(--interactivo);
     outline: none;
 }
+
+.config-container input[type=file]{
+    width: 100% !important;
+    flex-basis: auto;
+    padding: 15px;
+    flex-grow: 3;
+    cursor: pointer;
+    background-color: white;
+}
+#seed{
+    width: 110px;
+}
+/*
+#seed {
+    width: 100%;
+    flex-basis: 100%;
+    display: inline-block;
+    flex-direction: row;
+}
+#reset {
+    font-size: 14px;
+    width: 20%;
+    flex-basis: 20%;
+    display: inline-block;
+    flex-direction: row;
+}
+*/
+
+/*---------------------BOTON-SUBMIT---------------------*/
+#submit, #reset {
+    width: 100%;
+    font-size: 16px;
+    margin-top: 20px;
+    background-color: var(--interactivo);
+    border-radius: 8px;
+    color: white;
+    padding: 10px;
+    outline: none;
+    border: none;
+    cursor: pointer;
+    /*background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);*/
+    background: linear-gradient(-45deg, #CC648F, #3C308C, #171640, #6387A6, #A0E3F2);
+	background-size: 200% 200%;
+}
 #submit {
-    border-radius: 0px 20px 20px 0px;
-    padding: 5px 10px 5px 10px;
-    border: 1px solid black;
+    padding: 10px 20px;
+}
+
+#submit:hover, #reset:hover{
+    background-color:rgb(3, 128, 201);
+	animation: gradient 1.1s ease infinite;
 }
-#results {
+
+@keyframes gradient {
+	0% {
+		background-position: 0% 50%;
+	}
+	50% {
+		background-position: 100% 50%;
+	}
+	100% {
+		background-position: 0% 50%;
+	}
+}
+
+/*---------------------ABOUT---------------------*/
+#about{
     text-align: center;
-    max-width: 1000px;
-    margin: auto;
-    padding-top: 10px;
+    margin-top: 20px;
+}
+/*-------------------UP--------------------------*/
+#up{
+    width: 100px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    margin: 10px 10px 0 0;
+    background-color: var(--interactivo);
+    border-radius: 8px;
+    color: white;
+    padding: 10px;
+    outline: none;
+    border: none;
+    cursor: pointer;
+    /*background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);*/
+    background-color:  #3C308C;
 }
+
 img {
     cursor: pointer;
-    height: 30vh;
     border-radius: 5px;
     margin: 10px;
 }
-#fieldset-config {
-    line-height:2em;
-}
+
 input[type="number"] {
     width: 60px;
 }
-#seed {
-    width: 150px;
+
+
+/*----------Responsive----------*/

```

Comment:
Should be all caps to keep with other comments

Reply:
Solved